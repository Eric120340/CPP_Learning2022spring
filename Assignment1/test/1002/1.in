4 1
64192 
49850 
76337 
11425 
4 1
3487 
44932 
848 
11536 
