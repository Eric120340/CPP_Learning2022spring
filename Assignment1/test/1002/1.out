67679 
94782 
77185 
22961 
